pilot/util/harvester.py

```diff
@@ -126,11 +132,14 @@ def get_event_status_file(args):
     :param args: Pilot arguments object.
     :return: event staus file name.
     """
+
+    logger.debug('config.Harvester.__dict__ : {0}'.format(config.Harvester.__dict__))
+
     if args.harvester_workdir != '':
         work_dir = args.harvester_workdir
     else:
         work_dir = environ['PILOT_HOME']
-    event_status_file = config.Harvester.StageOutnFile
```

Comment:
Ok to rename the field, but you need to update the default.cfg file as well then